webdriver-spec.html

```diff
@@ -4105,7 +4108,46 @@
 
  <li><p><a>Scroll into view</a> the <var>element</var>.
 
- <li><p class=issue>Interact with the element.
+ <li><p>Run the <a>focusing steps</a> for the <a>element</a>.
+
+ <li><p>Set the cursor position to after the last character in the element
+
+ <li><p>Let <var>character array</var> be the result of <a>getting a property</a>
+    <code>text</code> from the <var>parameters</var> argument.
+
+ <li><p>If <var>element</var> is a <a><code>input</code></a> whose
+   <a><code>type</code> attribute</a> is <a>File</a>:
+   <ol>
+     <li><p>Let <var>text</var> be the result of
+      joining together each element in <var>character array</var> to a string.
+
+     <li><p>Set the <a><code>value</code> Attribute</a> to <var>text</var>.
+
+     <li><p>Jump to the last step in this overall set of steps.
+   </ol>
+
+  <li><p>Let <var>i</var> be 0.
+
+  <li><p>Let <var>length</var> be the length of <var>character array</var>.
+
+  <li><p>While <var>i</var> &lt; <var>length</var>:
+
+    <ol>
+      <li><p>Let <var>char</var> be the entry in <var>character array</var> at
```

Comment:
Can we please not use iterators with indexes?  It’s so hard to read for no apparent purpose.